lib/shopify_cli/messages/messages.rb

```diff
@@ -42,7 +42,7 @@ module Messages
             invalid_type: "The type %s is not supported. The only supported types are"\
               " {{command:[ rails | node | php ]}}",
             help: <<~HELP,
-            {{command:%s app create}}: Creates a ruby on rails app.
+            {{command:%s app create}}: Creates a new project in a subdirectory.
```

Comment:
@MeredithCastile we were showing this when you just run `app create`:

```
$ shopify app create
shopify app create: Creates a ruby on rails app.
  Usage: shopify app create [ rails | node | php ]
```

But it could also be a Node or PHP app... Let me know if you prefer another description.

Reply:
Hmmm. Maybe this?

> Creates a basic app
> Usage: shopify app create [ rails | node | php ]

Follow up question — Is there a default? Or will the system default to rails?